src/neuroconv/tools/nwb_helpers/_dataset_configuration.py

```diff
@@ -131,11 +131,12 @@ def get_default_dataset_io_configurations(
                 yield dataset_io_configuration
         else:
```

Comment:
I wonder whethre we can use an elif here and check for:
* being a container
* having either data or timestamps as attributes.

Reply:
Tried it out: https://github.com/catalystneuro/neuroconv/pull/804/commits/f528b88be248ea44cae06ddffc897c488eedd5f5

Reply:
Can't really see if it worked because of the other failing tests I think. Did it?

Reply:
@h-mayorquin My bad, dumb mistake. Fixed now and tests passing

Reply:
Great.